generator/C/include_v2.0/mavlink_types.h

```diff
@@ -15,6 +15,10 @@ namespace mavlink {
 // Macro to define packed structures
 #ifdef __GNUC__
   #define MAVPACKED( __Declaration__ ) __Declaration__ __attribute__((packed))
+#elif defined __arm__
```

Comment:
![default](https://user-images.githubusercontent.com/1421408/27292284-97defc40-5545-11e7-85ac-fa1890f6907a.png)
Yes. I tried, and it works.